book.rb

```diff
@@ -0,0 +1,17 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
```

Comment:
- [ ] you should create a new instance of rentals, remember you already pushed to rental.book in your rental class your code should look like this `Rental.new(date, self, person)`